specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/privateLinkService.json

```diff
@@ -0,0 +1,536 @@
+        },
+        "x-ms-examples": {
+          "approve or reject private end point connection for a private link service": {
+            "$ref": "./examples/PrivateLinkServiceUpdatePrivateEndpointConnection.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "PrivateLinkServices"
+        ],
+        "operationId": "PrivateLinkServices_DeletePrivateEndpointConnection",
+        "description": "Delete private end point connection for a private link service in a subscription.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "serviceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the private link service."
+          },
+          {
+            "name": "peConnectionName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the private end point connection."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted and the operation will complete asynchronously."
+          },
+          "204": {
+            "description": "Delete successful."
+          },
+          "200": {
+            "description": "Delete successful."
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "./network.json#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "delete private end point connection for a private link service": {
+            "$ref": "./examples/PrivateLinkServiceDeletePrivateEndpointConnection.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "PrivateLinkService": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/PrivateLinkServiceProperties",
+          "description": "Properties of the private link service."
+        },
+        "etag": {
+          "type": "string",
+          "description": "Gets a unique read-only string that changes whenever the resource is updated."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/Resource"
+        }
+      ],
+      "description": "Private link service resource."
+    },
+    "PrivateLinkServiceProperties": {
+      "properties": {
+        "loadBalancerFrontendIPConfigurations": {
+          "type": "array",
+          "items": {
+            "$ref": "./loadBalancer.json#/definitions/FrontendIPConfiguration"
```

Comment:
Per description this is list of IDs, so should elements be of [SubResource](https://github.com/Azure/azure-rest-api-specs/blob/network-april-release/specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/network.json#L137) type?

Please also check for other references.

Reply:
Could you please clarify how these reference properties look like when payload received from server?
Per [example](https://github.com/Azure/azure-rest-api-specs/pull/5860/files#diff-f6318fb15fde306b76e32a8cc9ae4e02R90), NetworkInterface reference contains **only Id** (if we reference full NIC it should contain all its properties), so we have specs-example mismatch there.
I think the best way to create specs examples is the following:
1. Call API using some tool that allows to do this directly, say, [ARMClient](https://github.com/projectkudu/ARMClient)
2. Copy resulting JSON payload to example so it exactly describes what we get from server
3. Check if API specs match server payload and adjust them, if necessary.

Reply:
Hi Evgeny, thank you for your reviewing :)  Let me ask the designer of those API to double check whether they need only Id or the whole object. And then I can change it accordingly.